CMakeLists.txt

```diff
@@ -383,7 +383,7 @@ endif()
 
 if(RDK_USE_BOOST_SERIALIZATION)
     set(T_LIBS ${Boost_LIBRARIES})
-    find_package(Boost ${RDK_BOOST_VERSION} COMPONENTS serialization)
+    find_package(Boost ${RDK_BOOST_VERSION} COMPONENTS system  serialization iostreams)
```

Comment:
I had to make this change to get my local build to work. Should I take it out of the commit?